application/templates/static_site/measure.html

```diff
@@ -100,9 +100,35 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l" itemprop="headline">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-4  govuk-!-margin-top-2" itemprop="headline">
         {{ measure_version.title }}
       </h1>
+
+      {%  if measure_version.first_published_date %}
+        <p class="govuk-body">Published <time itemprop="datePublished" datetime="{{ measure_version.first_published_date | format_iso8601_date }}">{{ measure_version.first_published_date | format_friendly_date }}</time>
+        </p>
+      {%  endif %}
+
+      <div class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</div>
+      <ol class="govuk-list eff-list__table-of-contents govuk-!-margin-bottom-9">
+        <li><a class="govuk-link" href="#main-facts-figures" data-on="click" data-event-category="Table of contents link clicked" data-event-action="Main facts and figures" data-event-label=""><span class="eff-list__table-of-contents__number">1. </span><span class="eff-list__table-of-contents__content">Main facts and figures</span></a></li>
+
+
+        {% if measure_version.dimensions %}
+          {% for dimension in measure_version.dimensions %}
+
+            <li><a class="govuk-link" href="#{{ dimension.title|slugify_value }}"><span class="eff-list__table-of-contents__number" data-on="click" data-event-category="Table of contents link clicked" data-event-action="Dimension" data-event-label="{{ dimension.title }}">{{ loop.index + 1 }}. </span><span class="eff-list__table-of-contents__content">{{ dimension.title }}</span></a></li>
+
+
+          {% endfor %}
+        {% endif %}
+
+
+        <li><a class="govuk-link" href="#methodology"><span class="eff-list__table-of-contents__number" data-on="click" data-event-category="Table of contents link clicked" data-event-action="Methodology" data-event-label="">{{ measure_version.dimensions.all()|length + 2 }}. </span><span class="eff-list__table-of-contents__content">Methodology</span></a></li>
```

Comment:
Might be nice to store the count of dimensions in a (temporary-ish) Jinja var here, so that we don't need to do `measure_version.dimensions.all()|length` multiple times?

E.g.
```Jinja2
{% set num_dimensions = measure_version.dimensions.all()|length %}

<li>
  <a class="govuk-link" href="#methodology">
    <span class="eff-list__table-of-contents__number" data-on="click" data-event-category="Table of contents link clicked" data-event-action="Methodology" data-event-label="">
      {{ num_dimensions + 2 }}. 
    </span>
    <span class="eff-list__table-of-contents__content">
      Methodology
    </span>
  </a>
</li>
```